src/Operation/Watch.php

```diff
@@ -173,11 +174,10 @@ public function __construct(Manager $manager, $databaseName, $collectionName, ar
         }
 
         $options += [
-            'fullDocument' => self::FULL_DOCUMENT_DEFAULT,
             'readPreference' => new ReadPreference(ReadPreference::RP_PRIMARY),
         ];
 
-        if (! is_string($options['fullDocument'])) {
+        if (array_key_exists('fullDocument', $options) && ! is_string($options['fullDocument'])) {
```

Comment:
array_key_exists function is used to avoid case when array field "fullDocument" with NULL value is passed from input (there is already exists an incorrect NULL-value input data test in MongoDB\Tests\Operation\WatchTest:55)